lib/modules/showImages.js

```diff
@@ -3072,9 +3072,9 @@ modules['showImages'] = {
 		},
 		steampowered: {
 			name: 'steam',
-			domains: ['steampowered.com'],
+			domains: ['steampowered.com', 'steamusercontent.com'],
 			detect: function(href, elem) {
-				return (/^cloud(?:-\d)?.steampowered.com$/i).test(elem.host);
+				return (/^cloud(?:-\d)?.(?:steampowered|steamusercontent).com$/i).test(elem.host);
```

Comment:
You should escape the periods so they don't match *every* character, like so: `\.`.